.github/workflows/main.yml

```diff
@@ -178,6 +178,14 @@ jobs:
         with:
           node: ${{ env.NODE_VERSION }}
           solana: ${{ env.SOLANA_VERSION }}
+      - name: Set environment variables
+        shell: bash
+        run: pnpm zx ./scripts/ci/set-variables.mjs
```

Comment:
You could add this script invocation to the `package.json` and just do `pnpm ci:set-env`.
Your call, just let me know if I need to update my repository.

https://github.com/paladin-bladesmith/rewards-program/blob/8934926b08280fd6f1abb1b8f17dba761f573f54/package.json#L4

Reply:
Perhaps it would be better not to add it on `package.json` since it is not something that needs to be "expose" to a user. What do you think?

Reply:
Yeah you're probably right. I'll update mine!